content/blog/overview-package-management-python.md

```diff
@@ -0,0 +1,363 @@
++++
+title = "An overview of package management in Python in 2018"
+description = "There are tons of things happening in package management for Python in 2018!"
```

Comment:
Not sure if "tons of things" is a good SEO term ^^

Reply:
Also, you're only talking about pipenv and pip-tools, which are used in conjunction. not my definition of tons :stuck_out_tongue:

Reply:
oh, I had missed poetry :)

Reply:
pip-tools and pipenv are not used in conjunction?

Reply:
Why would pipenv try to clear pip-tools cache when given the `--clear` parameter then ? sorry, haven't looked at it in a while, am probably wrong  :)

Reply:
Hmm I haven't mentioned cache in the pipenv part. I just said that pipenv uses pip-tools for some of the commands under the hood

Reply:
Roger it, makes sense :)